ui/src/containers/InventoryLayout.js

```diff
@@ -2,6 +2,7 @@ import * as inventoryDuck from '../ducks/inventory'
 import * as productDuck from '../ducks/products'
 import Checkbox from '@material-ui/core/Checkbox'
 import Grid from '@material-ui/core/Grid'
+import InventoryFormModal from '../components/Inventory/InventoryFormModal'
```

Comment:
Did you forget to push something up? This file doesn't exist in this PR

Reply:
My bad, I pushed it just now.